src/Microsoft.Graph.Core/Requests/DeltaResponseHandler.cs

```diff
@@ -0,0 +1,116 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph
+{
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// PREVIEW 
+    /// A response handler that exposes the list of changes returned in a response.
+    /// This supports scenarios where the service expresses changes to 'null'. The
+    /// deserializer can't express changes to null so you can now discover if a property
+    /// has been set to null. This is intended for use with a Delta query scenario.
+    /// </summary>
+    public class DeltaResponseHandler : IResponseHandler
```

Comment:
Add docs on how to use this. Ideally, this is only used after an initial sync, as we have to inspect the contents of the response twice to determine the change list.